rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/console/TestExecutionLogViewPart.xtend

```diff
@@ -26,7 +26,9 @@ class TestExecutionLogViewPart {
 	@PostConstruct
 	def void buildUI(Composite parent) {
 		browser = new Browser(parent, SWT.NORMAL)
-		browser.setUrl("http://localhost:19091/testlogs/web-app/index.html")
+		val url = "http://localhost:" + System.getProperty("org.osgi.service.http.port") +
```

Comment:
I used the osgi Property to Reuse the existing osgi Server. If we have a custum Port we should also manage a separate Server instance. 
For rcp there is in another branch an aproach to select a Free Port. For an Developer in the ide i prefer to let him manage it with thr Default Property.
Further work could be a config to a Remote Server.

I will fix the sugggested String building Way